docs/modules/ROOT/partials/getting-started/con-registry-serdes-protobuf.adoc

```diff
@@ -5,43 +5,52 @@
 
 = Configure Protobuf SerDe with {registry}
 
-{registry} provides Kafka client serializer and deserializer classes for Google Protobuf to make using Protobuf as easy as possible:
+[role="_abstract"]
+{registry} provides the following Kafka client serializer and deserializer classes for Google Protobuf:
 
-* `io.apicurio.registry.utils.serde.ProtobufKafkaSerializer`
-* `io.apicurio.registry.utils.serde.ProtobufKafkaDeserializer`
+* `io.apicurio.registry.serde.protobuf.ProtobufKafkaSerializer`
+* `io.apicurio.registry.serde.protobuf.ProtobufKafkaDeserializer`
 
 .Configure the Protobuf serializer
 
-You can configure the Protobuf serializer class in the following ways:
+You can configure the Protobuf serializer class as follows:
 
-* {registry} location as a URL
-* Artifact ID strategy 
-* Global ID strategy 
+* {registry} URL
+* Artifact resolver strategy 
+* Global ID strategy  
 * Global ID location
 * Global ID handler
 
+For details on these configuration options, see the following sections: 
+
+* xref:registry-serdes-concepts-constants-registry[]
+* xref:registry-serdes-types-avro-registry[]
+
 .Configure the Protobuf deserializer
 
-You must configure the Protobuf deserializer class to match the configuration settings of the serializer.  As a result, you can configure the Protobuf deserializer class in the following ways:
+You must configure the Protobuf deserializer class to match the following configuration settings in the serializer:
 
 * {registry} location as a URL
 * Global ID handler
```

Comment:
global Id handler is not a configuration option for protobuf deserializer.
Besides that there are some new config properties , check `io.apicurio.registry.serde.protobuf.ProtobufKafkaDeserializerConfig`